js/utils.js

```diff
@@ -0,0 +1,8 @@
+function getRandomPositiveInteger (a, b) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+}
+const getRandomArrayElement = (elements) => elements [getRandomPositiveInteger(0, elements.length - 1)];
```

Comment:
так меньше букв нажимать в редакторе, а главное видно, какие функции "публичные"
```suggestion
export const getRandomArrayElement = (elements) => elements [getRandomPositiveInteger(0, elements.length - 1)];
```